cmd/build/helmify/static/templates/gatekeeper-mutating-webhook-configuration-mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,33 @@
+
+{{- if .Values.experimentalEnableMutation }}
```

Comment:
This file can be added: https://github.com/open-policy-agent/gatekeeper/tree/master/manifest_staging/charts/gatekeeper/templates